packages/fake-data/src/fakeUserSkills.ts

```diff
@@ -0,0 +1,61 @@
+import { faker } from "@faker-js/faker";
+import { UniqueEnforcer } from "enforce-unique";
+
+import {
+  Skill,
+  SkillLevel,
+  UserSkill,
+  WhenSkillUsed,
+} from "@gc-digital-talent/graphql";
+
+import fakeUsers from "./fakeUsers";
+import { getStaticSkills } from "./fakeSkills";
+import { AnyGeneratedExperience } from "./fakeExperiences";
+
+const staticSkills = getStaticSkills();
+const randomSkill = faker.helpers.arrayElement<Skill>(staticSkills);
+const mockUser = fakeUsers(1)[0];
+
+const generateUserSkill = (
+  skill: Skill,
+  experiences: AnyGeneratedExperience[],
+  uniqueEnforcerId: UniqueEnforcer,
+) => {
+  const uniqueId = uniqueEnforcerId.enforce(() => {
+    return faker.string.uuid();
+  });
+  return {
+    __typename: undefined,
+    id: uniqueId,
+    skill,
+    user: mockUser,
```

Comment:
I reccomend passing user in as an argument, with a default value of mockUser

Reply:
Sounds good, I added it as an arg and set a default incase we don't care who the user is.

[improve fake user skill generation](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7733/commits/16a0f8fc5c1554e77b89e2ab22f99e60d521c47c)